src/bindings/python/src/openvino/frontend/jax/utils.py

```diff
@@ -0,0 +1,124 @@
+
+from openvino.runtime import op, Type as OVType, Shape, OVAny
+
+def get_type_from_py_type(value):
+    if isinstance(value, float):
+        return OVType.f32
+    if isinstance(value, bool):
+        return OVType.boolean
+    if isinstance(value, int):
+        return OVType.i64
+    return OVType.dynamic
+
+def get_ov_type_for_value(value):
+    if isinstance(value, (jax.core.Var, jax.core.Literal)):
+        if value.aval.dtype in jax_to_ov_type_map:
+            return OVAny(jax_to_ov_type_map[value.aval.dtype])
+        for k, v in numpy_to_ov_type_map.items():
+            if isinstance(value.aval.dtype, k):
+                return OVAny(v)
+        for k, v in basic_to_ov_type_map.items():
+            if isinstance(value.aval.dtype, k):
+                return OVAny(v)
+    elif isinstance(value, (int, float, bool)):
+        return OVAny(jax_to_ov_type_map[type(value)])
+    else:
+        raise NotImplementedError(f"dtype for {value} of type {type(value)} has not been supported yet.")
+    
+def get_ov_type_from_jax_type(dtype):
+    if dtype in jax_to_ov_type_map:
+        return OVAny(jax_to_ov_type_map[value.aval.dtype])
+    for k, v in numpy_to_ov_type_map.items():
+        if isinstance(dtype, k):
+            return OVAny(v)
+    for k, v in basic_to_ov_type_map.items():
+        if isinstance(dtype, k):
+            return OVAny(v)
+    return None
+
+def jax_array_to_ov_const(arr: np.ndarray, shared_memory=True):
+    # TODO: deal with bfloat16 dtype here.
+    if isinstance(arr, np.ndarray):
+        return op.Constant(arr, shared_memory=shared_memory)
+    elif isinstance(arr, jax.Array):
+        return op.Constant(np.array(jax.device_get(arr)), shared_memory=shared_memory)
+    else:
+        raise ValueError(f"Constant is expected to be a numpy array or jax array but got {type(arr)}")
+
+def ivalue_to_constant(ivalue, shared_memory=True):
+    ov_type = get_type_from_py_type(ivalue)
+    if ov_type.is_static():
+        return ov_type, Shape([]), op.Constant(ov_type, Shape([]), [ivalue]).outputs()
+
+    if isinstance(ivalue, (list, tuple)):
+        assert len(ivalue) > 0, "Can't deduce type for empty list"
+        ov_type = get_type_from_py_type(ivalue[0])
+        assert ov_type.is_static(), "Can't deduce type for list"
+        return ov_type, Shape([len(ivalue)]), op.Constant(ov_type, Shape([len(ivalue)]), ivalue).outputs()
+
+    if isinstance(ivalue, (jax.Array, np.ndarray)):
+        return get_ov_type_from_jax_type(ivalue.dtype), Shape(ivalue.shape), \
+            jax_array_to_ov_const(ivalue, shared_memory=shared_memory).outputs()
+            
+    ov_dtype_value = get_ov_type_from_jax_type(ivalue)
+    if ov_dtype_value is not None:
+        return ov_dtype_value, Shape([]), \
+            op.Constant(OVType.i64, Shape([]), [ov_type_to_int_map[ov_dtype_value]]).outputs()
+    
+    print(f"[WARNING][JAX FE] Cannot get constant from value {ivalue}")
+    return OVAny(None), None, None
```

Comment:
please return just constant node here because you can get its type and shape easily.